packages/server-wallet/src/message-service/types.ts

```diff
@@ -22,6 +22,12 @@ export interface MessageServiceInterface {
    */
   send(message: Message[]): Promise<void>;
 
+  // TODO: Is a url/string too restrictive for addressing a peer?
+  /**
+   * Registers a peer so messages can be sent ot that peer
+   * @param peerUrl The url for the peer
+   */
+  registerPeer(peerUrl: string): Promise<void>;
```

Comment:
(This applies to the doc comment above as well)

I think that this function subscribes to messages emitted by the given peer, and does NOT register the peer for *sending* messages. Is `subscribeToPeer` a better name?

A question: can the server objects here (`import Server from 'socket.io'` listen for new connections and then trigger a subscription to messages from peers that connect?

Reply:
`RegisterPeer` actually does both. It registers a listener for when a message is received from the peer but it also adds the peer to the peer list so the peer will be sent messages.

> A question: can the server objects here (import Server from 'socket.io' listen for new connections and then trigger a subscription to messages from peers that connect?

Possibly! However, since we need something like `registerPeers` to register peers to send messages to I'm not sure if that cleans things up much.